python/Ganga/new_tests/GPI/Bugs/Issue185.py

```diff
@@ -0,0 +1,31 @@
+from ..GangaUnitTest import GangaUnitTest
+
+class Issue185(GangaUnitTest):
```

Comment:
You're right, it was just running over `new_tests/GPI` but I've just changed it so that it runs over all of `new_tests` so this will include all unit tests too. I'm not sure why I hadn't set it up like that before.